core/src/main/scala/org/apache/spark/TestUtils.scala

```diff
@@ -236,7 +236,11 @@ private[spark] object TestUtils {
    * Test if a command is available.
    */
   def testCommandAvailable(command: String): Boolean = {
-    val attempt = Try(Process(command).run(ProcessLogger(_ => ())).exitValue())
+    val attempt = if (Utils.isWindows) {
+      Try(Process(s"WHERE $command").run(ProcessLogger(_ => ())).exitValue())
```

Comment:
I run some tests via AppVeyor manually to show it works in Windows:

Build started: [CORE] `org.apache.spark.rdd.PipedRDDSuite` [![PR-29241](https://ci.appveyor.com/api/projects/status/github/HyukjinKwon/spark?branch=26712025-86F9-40C2-BF08-1E4BD683B021&svg=true)](https://ci.appveyor.com/project/HyukjinKwon/spark/branch/26712025-86F9-40C2-BF08-1E4BD683B021)

Build started: [SQL] `org.apache.spark.sql.hive.execution.SQLQuerySuite` [![PR-29241](https://ci.appveyor.com/api/projects/status/github/HyukjinKwon/spark?branch=07527AA5-8FF9-4E05-BA66-788B54BD8A26&svg=true)](https://ci.appveyor.com/project/HyukjinKwon/spark/branch/07527AA5-8FF9-4E05-BA66-788B54BD8A26)

Some of tests might already fail. We can just see if `testCommandAvailable` works or not.